src/state-summary/state-card-display.html

```diff
@@ -16,12 +16,17 @@
         @apply(--layout-baseline);
       }
 
+      state-info {
+        flex: 1 1 auto;
+        min-width: 0;
+      }
       .state {
         @apply --paper-font-body1;
         color: var(--primary-text-color);
         margin-left: 16px;
         text-align: right;
-        flex: 1 0 50px;
+        max-width: 40%;
```

Comment:
That's a waste when the name is short.

Reply:
Yes, but it prevents very long states to push the name away.
I could set it to 50%, that leave about 20% for the name (the icon takes quite some space too)

The point is here that the state can take it's space with a max of 40%, and the state-info takes the rest.

Reply:
I've updated the PR description with some screenshots.

Reply:
So it means that we prefer state info to take more space over state controls, and for state controls that we need it, we'll add a `white-space: nowrap`? I guess that can work.

What about the cover controls?

Reply:
Yeah, it might be impacted by the cramped space. Just looked that it already has a nowrap set.

We should create a wiki page where we specify the following things: min width of a card, min space for name, min space for controls. I am also thinking that we might want to create a dev page that renders all different states of an entity at once, next to one another, at different sizes.

Reply:
The cover controls are fine currently. (although they take up quite some space)

A dev-page showing the different state cards with different names (long and short) and sizes would be an awesome tool to check style changes.
We could create a custom component that just spits out a variety of states and groups/views and a custom panel show those in some wide and cramped dimensions.